README.md

```diff
@@ -89,7 +89,13 @@ The [QuickStart](#quickstart) step above will install Zig for you to the root of
 To run TigerBeetle's long-running simulation, called *The VOPR*:
 
 ```bash
-scripts/vopr.sh
+zig/zig build vopr
+```
+
+Pass the `send` flag if you want the VOPR also to report any bugs to the VOPR Hub, which can replay, deduplicate, and automatically create GitHub issues as needed.
+
+```bash
+zig/zig build vopr -- --send
```

Comment:
It would be great to also have an example here in the README of running a specific seed or getting help to see all the available command line arguments.